test/e2e/parallel/registration_service_test.go

```diff
@@ -273,94 +278,100 @@ func (s *registrationServiceTestSuite) TestSignupFails() {
 		emailClaim := authsupport.WithEmailClaim(emailValue)
 		usernameClaim := authsupport.WithPreferredUsernameClaim("test-crtadmin")
 		token, err := authsupport.GenerateSignedE2ETestToken(*identity, emailClaim, usernameClaim)
-		require.NoError(s.T(), err)
+		require.NoError(t, err)
 
 		// Call signup endpoint with a valid token to initiate a signup process
-		response := invokeEndpoint(s.T(), "POST", s.route+"/api/v1/signup", token, "", http.StatusForbidden)
-		require.Equal(s.T(), "forbidden: failed to create usersignup for test-crtadmin", response["message"])
-		require.Equal(s.T(), "error creating UserSignup resource", response["details"])
-		require.Equal(s.T(), float64(403), response["code"])
-
-		hostAwait := s.Host()
-		userSignup, err := hostAwait.WaitForUserSignup(identity.ID.String())
-		require.Nil(s.T(), userSignup)
-		require.Error(s.T(), err)
-		require.EqualError(s.T(), err, "timed out waiting for the condition")
+		response := invokeEndpoint(t, "POST", route+"/api/v1/signup", token, "", http.StatusForbidden)
+		require.Equal(t, "forbidden: failed to create usersignup for test-crtadmin", response["message"])
+		require.Equal(t, "error creating UserSignup resource", response["details"])
+		require.Equal(t, float64(403), response["code"])
+
+		hostAwait := await.Host()
+		hostAwait.WaitAndVerifyThatUserSignupIsNotCreated(identity.ID.String())
```

Comment:
We usually use custom (shorter) timeouts for checks which verify that the resource does **not** exist to speed up the test execution. An example: https://github.com/codeready-toolchain/toolchain-e2e/blob/c288b650eb6f8df5cbb6800f59bbc0b1ebae3856/test/metrics/metrics_test.go#L241

So something like 
```suggestion
		hostAwait.WithRetryOptions(wait.TimeoutOption(time.Second*15)).WaitAndVerifyThatUserSignupIsNotCreated(identity.ID.String())
```
The default timeout is 60 seconds. So otherwise this test will hang for 60 seconds if we do not use a custom timeout.

Reply:
Ah, sure - just forgot to add the shorter timeout again when solving the conflict locally.
Good catch - thanks :+1: